### DISCUSSION TITLE
Remove the horizonal scrollbar on ScrollView

### DISCUSSION THREAD

**Discussion by ethansocal** [Textualize/textual#30] [Q&A]
How can I remove the horizontal scrollbar on ScrollView? The latest update seems to have added it.

**Comment by willmcgugan** [comment]
Eventually it will appear automatically if needed, in the meantime you could do this:

```python
scroll_view._horizontal_scrollbar.visible = False
```

**Reply by willmcgugan** [reply]
Actually, this may be better:

```python
scroll_view.layout.hide_row("horizontal")
```

**Comment by willmcgugan** [comment]
0.1.7 should hide the horizontal scrollbar automatically
